packages/database/src/migrations/20210714053110-FirstNationWithILIGraph-modifies-data.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const reportCode = 'AU_FLUTRACKING_First_Nation_With_ILI';
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+exports.up = async function (db) {
+  const dashboardItemId = generateId();
+  const dashboardRelationId = generateId();
+  await insertObject(db, 'report', {
+    id: generateId(),
+    code: reportCode,
+    config: {
+      fetch: {
+        aggregations: ['RAW'],
+        dataElements: ['FluTracker_LGA_Percent_First_Nations_ILI'],
+      },
+      transform: [
+        'convertPeriodToWeek',
+        {
+          transform: 'select',
+          "'name'": '($row.period).substring(4, 7)',
+          '($row.period).substring(0, 4)': '$row.value',
+        },
+        {
+          transform: 'aggregate',
+          name: 'group',
+          '...': 'last',
+        },
+        {
+          transform: 'select',
+          "'timestamp'": "periodToTimestamp(('2020').concat($row.name))",
```

Comment:
Hackly adding a timestamp, so that we can order the xAxis.

Reply:
Defaulting to name order if no timestamp sounds like a good idea.
So 2020 is just an arbitrary date to create a weekly order?

Reply:
@nick-beyondessential That's right, yeah I incline that we don't always need to put a timestamp here just for the sorting, I would say that if they need a customised sorting they should do that like `stackOrder` in `chartConfig`.